tests/workflows/test_pytorch_optuna.py

```diff
@@ -3,25 +3,30 @@
 import tempfile
 import zipfile
 
-import optuna
 import pytest
-import torch
-import torch.nn as nn
-import torch.nn.parallel
-import torch.optim as optim
-import torch.utils.data
-from dask.distributed import PipInstall, wait
+from dask.distributed import PipInstall
+
+from ..utils_test import wait
+
+optuna = pytest.importorskip("optuna")
 
 
 @pytest.mark.client(
     "pytorch_optuna",
     worker_plugin=PipInstall(
         [
+            # FIXME package_sync doesn't seem to deduce GPU specific installs of
+            #       libraries like torch
             "torch==2.0.0",
-            "torchvision",
-            # FIXME: https://github.com/boto/botocore/issues/2926
-            # urllib3 v2 removed openssl / ciphers which causes an error in botocore httpsession
+            # FIXME https://github.com/coiled/benchmarks/pull/787#issuecomment-1532882045
+            #       Windows package_sync doesn't like torchvision
+            "torchvision==0.15.1",
+            # FIXME https://github.com/boto/botocore/issues/2926
+            #       urllib3 v2 removed openssl / ciphers which causes an error in
+            #       botocore httpsession
             "urllib3<2.0.0",
+            # FIXME https://github.com/optuna/optuna/pull/4589
+            #       Need optuna >3.1.1
```

Comment:
@milesgranger here you are installing a version of optuna on the client and a different one on the workers. I guess that the client-side part is luckily compatible with the worker-side one?

Reply:
At the time it was the same IIRC, this change was due to https://github.com/coiled/benchmarks/pull/787/files#r1196779222